src/components/pages/HomeView/LearnMore.tsx

```diff
@@ -0,0 +1,69 @@
+import { useIntl } from "react-intl";
+import { motion } from "framer-motion";
+import ContentBox from "./ContentBox";
+
+const LearnMore = () => {
+  const i18n = useIntl();
+
+  const learnMore = i18n.formatMessage({
+    id: "HomeView.LearnMore",
+    description: "",
+    defaultMessage: "Learn More",
+  });
+
+  return (
+    <motion.div
+      initial="hidden"
+      whileInView="visible"
+      viewport={{ amount: 0.3, once: true }}
+      variants={{
+        hidden: {
+          y: 80,
+          opacity: 0,
+        },
+        visible: {
+          y: 0,
+          opacity: 100,
+        },
+      }}
+      transition={{ duration: 0.8 }}
+      className="grid grid-cols-2 gap-8 py-40 md:grid-cols-1"
+    >
+      <ContentBox
+        title="Developer-Friendly"
```

Comment:
Should be a localised message. Same as for the description.
And same for the other box "Decentralized"

Reply:
Resolved